### PR TITLE
[docs] Some fixes to Content guides

### PR DESCRIPTION
**Pull request by sheila-semrush**
## Motivation and Context

Added various improvements to the Punctuation and Date guides.
Also added both of them to the left navigation menu (you're welcome to propose another order).
I think we're close to publishing them, let's look at them and decide.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Nice improve.

## Checklist:

- [ ] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Comment by j-mnizhek** [comment]
Looks good 😎 It's great that we can already show them to everyone ✨

I've only moved Content folder after Layout, because Style and Layout both about the appearance, and it will be good to keep the together I think.

File: website/docs/content/punctuation/punctuation.md [hunk 1]
```diff
@@ -203,12 +195,14 @@ Use ⌥+' on MacOS to insert a curly apostrophe
 
 <DosDonts>
     <template #dont>
-        <p>☑️ Remind later.</p>
-        <p>Your competitor https://rival.com.</p>
+        <img src="static/period-dont.png" style="margin: auto" />
```
Comment (j-mnizhek) [inline website/docs/content/punctuation/punctuation.md:198]: There is something strange with this particular image. I always get this error when trying to `pnpm build` the website or just open the Punctuation page:
image

**Review by j-mnizhek — APPROVED** [review]
All is good 😎

_I left only one comment about the error that ruins the website. Maybe something with paths, but not sure 🤔_ Need to check and fix before merging, of course.

File: website/docs/content/punctuation/punctuation.md (hunk 1 above)
Comment (sheila-semrush) [reply website/docs/content/punctuation/punctuation.md:198]: Must be an issue in your local setup, it builds without errors on my system, and the cd/build job on github also runs successfully.

**Pull request merged**